agent/exec/containerd/executor.go

```diff
@@ -0,0 +1,107 @@
+package containerd
```

Comment:
Let's call the `swarmkit/agent/exec/container` one `dockerapi`.

Would you mind doing that in a separate PR?